manifests/params.pp

```diff
@@ -1,7 +1,7 @@
 # @summary The default parameters for the foreman proxy
 # @api private
 class foreman_proxy::params inherits foreman_proxy::globals {
-  $tftp_root = lookup('tftp::root')
+  $tftp_root = lookup('tftp::root', undef, undef, '/not/used')
```

Comment:
Not sure it really adds much, but you could use an actual data type for the second argument instead of `undef`.
```suggestion
  $tftp_root = lookup('tftp::root', Stdlib::Unixpath, undef, '/not/used')
```